### PR TITLE
[ticket/16121] Footer classes in simple_footer

### PR DESCRIPTION
**Pull request by mrgoldy**
PHPBB3-16121

Checklist:

- [x] Correct branch: master for new features; 3.3.x & 3.2.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master](https://area51.phpbb.com/docs/dev/master/development/coding_guidelines.html), [3.3.x](https://area51.phpbb.com/docs/dev/3.3.x/development/coding_guidelines.html) and [3.2.x](https://area51.phpbb.com/docs/dev/3.2.x/development/coding_guidelines.html)
- [x] Commit follows commit message [format](https://area51.phpbb.com/docs/dev/3.3.x/development/git.html)

Tracker ticket (set the ticket ID to **your ticket ID**):

https://tracker.phpbb.com/browse/PHPBB3-16121

Adding same classes introduced in #5231 for overall_footer

### CODE REVIEW

**Comment by mrgoldy** [comment]
As I can't explicitly request a review, I'll just do it through a comment: @hanakin

**Comment by vinny** [comment]
[adm/style/simple_footer.html#L5~L17](https://github.com/phpbb/phpbb/blob/master/phpBB/adm/style/simple_footer.html#L5~L17) should also be updated

**Comment by mrgoldy** [comment]
> [adm/style/simple_footer.html#L5~L17](https://github.com/phpbb/phpbb/blob/master/phpBB/adm/style/simple_footer.html#L5~L17) should also be updated

Well, a couple of things with that:
- It's not front-end
- It does not have any of the `footer-` classes yet
- It can not be customised by styles
- Entire adm style needs updating
So not sure if that should be part of this simple PR?

File: phpBB/styles/prosilver/template/simple_footer.html [hunk 1]
```diff
@@ -1,8 +1,19 @@
...
-	<div class="copyright" role="contentinfo">{CREDIT_LINE}
-		<!-- IF TRANSLATION_INFO --><br />{TRANSLATION_INFO}<!-- ENDIF -->
-		<!-- IF DEBUG_OUTPUT --><br />{DEBUG_OUTPUT}<!-- ENDIF -->
+	<div class="copyright" role="contentinfo">
+		<p class="footer-row">
+			<span class="footer-copyright">{{ CREDIT_LINE }}</span>
+		</p>
+		{% if TRANSLATION_INFO %}
+			<p class="footer-row">
+				<span class="footer-copyright">{{ TRANSLATION_INFO }}</span>
```
Comment (Crizz0) [inline phpBB/styles/prosilver/template/simple_footer.html]: Same class as the Credit_line in Line 5? Maybe a "footer-translation-info" would be more sufficient.

Comment (mrgoldy) [reply phpBB/styles/prosilver/template/simple_footer.html]: This class is identical to the class used in the `overall_footer.html`: 
https://github.com/phpbb/phpbb/blob/master/phpBB/styles/prosilver/template/overall_footer.html#L16
So I would in favor of keeping the same class as there.

Comment (Crizz0) [reply phpBB/styles/prosilver/template/simple_footer.html]: Yeah, then better fix both later in a new PR.

**Pull request merged**